calculus.py

```diff
@@ -370,354 +327,3 @@ def trapezoid_scipy(func, l_lim, u_lim, steps=10000):
     y = func(x)    # evaluate the function on the modified grid
     integral_value = sp.integrate.trapezoid(y, x)   # calculate the integral using numpy
     return integral_value
-
```

Comment:
Why are these functions being deleted?

Reply:
the undefined pylint errors come from you still using d1,2,3 below but deleting them above. Can be fixed by not deleting them.